lib/elixir/lib/module.ex

```diff
@@ -543,6 +543,101 @@ defmodule Module do
   @callback __info__(:md5) :: binary()
   @callback __info__(:module) :: module()
 
+  @doc """
+  Returns information about module attributes used by Elixir.
+
+  See the "Module attributes" section in the module documentation for more
+  information on each attribute.
```

Comment:
I suppose this sentence is no longer true for `@derive` and `@enforce_keys`, how should we reword it? Or just remove it?

Reply:
Let’s add a quick sentence at the top on derive and enforce keys and point people to structs. :)